skymp5-server/cpp/server_guest_lib/MpActor.cpp

```diff
@@ -460,7 +460,7 @@ void MpActor::RespawnWithDelay(bool shouldTeleport)
 
   uint32_t formId = GetFormId();
   if (auto worldState = GetParent()) {
-    worldState->SetTimer(GetRespawnTime())
+    worldState->SetTimer(8.0f)
```

Comment:
Out of curiosity, why did you decide to hardcode this?

Reply:
Uh. This wasn't intentional. I was trying to fix a bug with health regeneration, and thought that it may have been related.
When I did fix it, I guess I accidently left it in.